modules/javafx.swing/src/test/java/test/javafx/embed/swing/SwingFXUtilsTest.java

```diff
@@ -43,46 +43,17 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+
```

Comment:
Your proposed fix for this test class is not the right fix, and needs to be reverted. It highlights an existing bug with the swing tests, which I will address in general comments.

Reply:
It's now removed